packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -276,25 +296,49 @@ Datepicker.propTypes = {
   showWeekNumbers: PropTypes.bool,
   workWeekDays: PropTypes.arrayOf(PropTypes.oneOf(Object.keys(DayOfWeek).map(name => DayOfWeek[name]))),
 
-  localizedStrings: PropTypes.object as PropTypes.Validator<IDateGridStrings>,
+  formatDay: PropTypes.func,
+  formatYear: PropTypes.func,
+  formatMonthDayYear: PropTypes.func,
+  formatMonthYear: PropTypes.func,
 
-  format: PropTypes.func,
-  parse: PropTypes.func,
+  parseDate: PropTypes.func,
 
-  disabled: PropTypes.bool,
-  isRequired: PropTypes.bool,
-  onDateChange: PropTypes.func,
+  months: PropTypes.arrayOf(PropTypes.string),
+  shortMonths: PropTypes.arrayOf(PropTypes.string),
+  days: PropTypes.arrayOf(PropTypes.string),
+  shortDays: PropTypes.arrayOf(PropTypes.string),
+
+  isRequiredErrorMessage: PropTypes.string,
+  invalidInputErrorMessage: PropTypes.string,
+  isOutOfBoundsErrorMessage: PropTypes.string,
   goToToday: PropTypes.string,
-  placeholder: PropTypes.string,
+  prevMonthAriaLabel: PropTypes.string,
```

Comment:
Why all of these are not in `LocalizedStrings`?

Reply:
@layershifter Do I understand that I should rather add the localizedStrings type as property and not extend the localizedStrings type?

Reply:
Oh, I see. All strings are props actually... Let's discuss this offline